crates/router/src/core/payments/flows.rs

```diff
@@ -2574,6 +2602,7 @@ impl<const T: u8>
 {
 }
 default_imp_for_connector_authentication!(
+    connector::Payone,
```

Comment:
Please add the connector enum in alphabetical order in all other places.

Reply:
done